ui/src/components/Card/Card.module.css

```diff
@@ -1,16 +1,18 @@
+@import "../../assets/fonts/fonts.module.css";
+@value colors "../../assets/styles/theme.module.css";
+@value main-light-mode from colors;
+
 .container {
     background-color: white;
     border-radius: 2px;
-    padding: 10px;
-    margin-top: 16px;
 }
 
-.container :global(.ant-table .ant-table-thead > tr > th) {
-    padding: 11px 16px;
-    font-weight: 500;
-    color: rgba(0, 0, 0, 0.85);
-    background: #fff;
-    border-bottom: 1px solid #595959;
+.widgetTitle {
+    color: main-light-mode;
+    font-family: openSans_bold;
```

Comment:
Fonts handling seems wrong; why do we have different declarations for the same font, and then assign fonts on per-component basis?

Usually it works like this:
1) One font-family on global level, components inherit font-family from the root and should not assign font-family explicitly unless it's that one place in layout where fonts are different from the entire app
2) Components with different variation of the root font use `font-weight`

Reply:
@Dergash fixed